arctic-report-card/ARCRC.ttl

```diff
@@ -232,508 +262,647 @@ Find current and past assessments of the Arctic Report Card Vital Signs by visit
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Frostbite, 'Working with Communities to Observe the Arctic'" ;
+                   rdfs:label "indigenous observations"@en .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000021
+odo:ARCRC_00000021 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "A measurement or estimate of the number of salmon, based on counts, densities, weights, or other methods." ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Frostbite, 'Western Alaska Salmon'" ;
+                   rdfs:label "salmon abundance"@en ;
+                   owl:sameAs "http://purl.dataone.org/odo/SALMON_00000504" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000022
+odo:ARCRC_00000022 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "A measurement of number of salmon swimming upstream to spawn" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Frostbite, 'Western Alaska Salmon'" ;
+                   rdfs:label "salmon run size"@en .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000023
+odo:ARCRC_00000023 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "The number of recruits which survive to a legal age and/or size and are captured by a fishery." ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Frostbite, 'Western Alaska Salmon'" ;
+                   rdfs:label "salmon harvest count"@en ;
+                   owl:sameAs "http://purl.dataone.org/odo/SALMON_00000491" .
+
+
+###  http://purl.dataone.org/odo/ARCRC_00000024
+odo:ARCRC_00000024 rdf:type owl:Class ;
+                   rdfs:subClassOf odo:ARCRC_00000040 ;
+                   obo:IAO_0000115 "The number of eggs a female salmon produces during a breeding season" ;
+                   dc:creator <http://orcid.org/0009-0007-8384-5074> ;
+                   dc:date "2024-07-30T19:43:23Z"^^xsd:dateTime ;
+                   dc:description "This Key Variable is used to inform Frostbite, 'Western Alaska Salmon'" ;
+                   rdfs:label "salmon fecundity"@en ;
+                   skos:relatedMatch "http://purl.dataone.org/odo/SALMON_00000622" .
```

Comment:
Is this the only relationship we can make?

Reply:
I added a narrowMatch to [Quality of a population measurement type](http://purl.dataone.org/odo/SALMON_00000476) which is defined as 

> Includes qualities like population size, population growth rate, carrying capacity, immigration rate, emigration rate, fecundity, and death rate. A population quality may depend on the qualities of individual organisms in the population, but cannot be measured or described for a single organism.